reservacionVuelo/Screens/MyFlights.js

```diff
@@ -0,0 +1,75 @@
+import React, {Component} from 'react';
+import {View, Text, TouchableHighlight, ListView, FlatList} from 'react-native';
+import styles from '../components/styles';
+import Icon from 'react-native-vector-icons/FontAwesome';
+import {FlightCard} from '../components/flightCard';
+
+const dummyData = [
+  {
+    id: 1,
+    departureCity: 'AGS',
+    departureCountry: 'MEXICO',
+    arrivalCity: 'LA',
+    arrivalCountry: 'USA',
+    year: 2021,
+    month: 'April',
+    day: 30,
+    passangers: 2,
+  },
+  {
+    id: 2,
+    departureCity: 'COL',
+    departureCountry: 'MEXICO',
+    arrivalCity: 'NY',
+    arrivalCountry: 'USA',
+    year: 2021,
+    month: 'April',
+    day: 22,
+    passangers: 3,
+  },
+  {
+    id: 3,
+    departureCity: 'MED ',
+    departureCountry: 'CDMX',
+    arrivalCity: 'BRL',
+    arrivalCountry: 'GERMANY',
+    year: 2021,
+    month: 'April',
+    day: 28,
+    passangers: 3,
+  },
+];
```

Comment:
As a suggestion, I recommend that you put this arrangement in another file, so that you don't overload the MyFlights.js file.